js/src/provider/WalletLinkProvider.ts

```diff
@@ -99,8 +99,13 @@ export class WalletLinkProvider
       ? options.walletLinkAnalytics
       : new WalletLinkAnalytics()
 
+    if (options.chainId) {
```

Comment:
Unrelated to this pr, but would please update `EthereumChain.ts` in `walletlink` to use `BN.js` instead of `bigint`. 

Using `bigint` was added fairly recently but it breaks walletlink for anyone using internet explorer.

Reply:
Will do. Once this version publishes, all consumers will udpate